scripts/select-random-player/select_random_player.lua

```diff
@@ -0,0 +1,30 @@
+local BE_SPECIAL = "Traitor"
+
+function selectRandomPlayer(be_special)
+
+    local players = dm.global_vars:get_var("GLOB"):get_var("alive_player_list")
+    local playerTable = {}
+
+    for _, player in players do
+        local client = player:get_var("client")
+        if not client then continue end
+        local be_special_client = client:get_var("prefs"):get_var("be_special")
+        for _, special in be_special_client do
+            if special == be_special then
+                table.insert(playerTable, player)
+                break
+            end
+        end
+    end
+
+    if #playerTable == 0 then 
+        dm.global_proc("message_admins", "Unable to select a player for the role of " .. be_special)
+        return 
+    end
+
+    local player = playerTable[math.random( #playerTable )]
+    dm.global_proc("message_admins", "Selected player " .. dm.global_proc("key_name_admin", player) .. " for the role of " .. be_special)
```

Comment:
Please test this for me real fast in tg:

```lua
print(`Answer = {2+2}`)
```

...string interpolation was added a long time ago, and so this should result in "Answer = 4". If that is the case on tg as well, then this should become:

```suggestion
    dm.global_proc("message_admins", `Selected player {dm.global_proc("key_name_admin", player)} for the role of {be_special}`)
```

Reply:
And if it doesn't work then we should poke Yoshi on updating the version of Luau we have

Reply:
I get an error message when testing `print(`Answer = {2+2}`)`

input errored with the message "src\lib.rs,553: syntax error: [string "input"]:1: Expected identifier when parsing expression, got '`'"